source/sdk/threading/Thread.ooc

```diff
@@ -165,6 +172,23 @@ Mutex: abstract class {
 
 }
 
+MutexUnsafe: class extends Mutex {
+	init: func
+	lock: override func
```

Comment:
We do, check in the other project.
Looks like it works kind of as `override` in C++, like a safety mechanism to make sure that you are implementing correct interface in the derived class. So if you provide an `override` specifier for a function that does not exist in the base class, you will get a compiler error.